stencils/pace/stencils/testing/grid.py

```diff
@@ -1089,3 +1089,93 @@ def _grid_indexing_axis_offsets(
     domain: Tuple[int, ...],
 ) -> Mapping[str, gtscript.AxisIndex]:
     return grid.axis_offsets(origin=origin, domain=domain)
+
+
+@dataclasses.dataclass(frozen=True)
+class DriverGridData:
```

Comment:
This is great, now we can retain the ability in translate test to read these from serialized data. Also, this should make it easier to add these variables in metric terms.